### PR TITLE
Smoke test check

### PR DESCRIPTION
**Pull request by sparrowDom**
Add an additional check that deploy script being smoke tested doesn't auto deploy on node startup.

Contract change checklist:
  - [ ] Code reviewed by 2 reviewers. See [template](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review.md) and [example](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md).
  - [ ] Unit tests pass
  - [ ] Slither tests pass with no warning
  - [ ] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

**Review by franckc — APPROVED** [review]
Thanks for adding this. LGTM

File: contracts/scripts/test/smokeTest.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # any child processes created by this process are killed once the main process is terminated
 trap "exit" INT TERM ERR
 trap "kill 0" EXIT
-nodeWaitTimeout=180
+nodeWaitTimeout=60
```
Comment (tomlinton) [inline contracts/scripts/test/smokeTest.sh:26]: Is this variable used?

Comment (sparrowDom) [reply contracts/scripts/test/smokeTest.sh:26]: Line 53 in this file. Maybe github collapsed a few lines of this file when showing the diff

Comment (tomlinton) [reply contracts/scripts/test/smokeTest.sh:26]: Haha yea my bad, must be half asleep, thanks :)

**Review by tomlinton — APPROVED** [review]

**Pull request merged**